requirements.txt

```diff
@@ -1,4 +1,2 @@
-cffi >= 0.8.0
-letsencrypt >= 0.7.0
-six >= 1.7
```

Comment:
this was for python2 compatibility and our goal was for it to run on either.  however, I'm not even sure it was used and now python2 is end-of-life, so I'm ok with this change